usaspending_api/recipient/v2/urls.py

```diff
@@ -2,10 +2,11 @@
 from usaspending_api.recipient.v2.views.states import StateMetaDataViewSet, StateAwardBreakdownViewSet, ListStates
 from usaspending_api.recipient.v2.views.recipients import RecipientOverView
 from usaspending_api.recipient.v2.views.recipients import ChildRecipients
-from usaspending_api.recipient.v2.views.list_recipients import ListRecipients
+from usaspending_api.recipient.v2.views.list_recipients import ListRecipients, RecipientCount
 
 urlpatterns = [
     url(r"^duns/$", ListRecipients.as_view()),
+    url(r"^duns/count/$", RecipientCount.as_view()),
```

Comment:
Again some inconsistency as to whether this API endpoint should be off of `recipient/count` or `recipient/duns/count`. If it is in fact just counting recipients and there's not implied filter by "duns", I would suggest it just live off of `recipient/count`. But if the `duns` moniker was added to imply that this endpoint allows you to search by duns ... then keep consistent with that I suppose.

Reply:
I have adjusted all these urls to match this format: `/api/v2/recipient/count/`